Dockerfile

```diff
@@ -17,36 +18,20 @@ RUN apk upgrade --update musl \
         openssl-dev \
         yaml-dev \
         expat-dev \
-        sqlite-dev \
-        gd-dev \
-        jpeg-dev \
-        libpng-dev \
-        libwebp-dev \
         autoconf \
         automake \
-        bash \
-        elixir \
-        erlang-dev \
-        erlang-crypto \
-        erlang-eunit \
-        erlang-mnesia \
-        erlang-erts \
-        erlang-hipe \
-        erlang-tools \
-        erlang-os-mon \
-        erlang-syntax-tools \
-        erlang-parsetools \
-        erlang-runtime-tools \
-        erlang-reltool \
-        file \
-        curl \
-        wget \
     && rm -rf /var/cache/apk/*
 
-# Install rebar
-WORKDIR /
-RUN wget https://s3.amazonaws.com/rebar3/rebar3 -O /usr/bin/rebar3
-RUN chmod +x /usr/bin/rebar3
+# Install rebar3
```

Comment:
If you're not installing rebar3 I prefer deleting lines to commenting them out

Reply:
Oops, sorry!